NFIQ2/NFIQ2Algorithm/src/NFIQ2Algorithm.cpp

```diff
@@ -35,6 +35,18 @@ NFIQ::NFIQ2Algorithm::computeQualityScore(
              qualityFeatureSpeed ) );
 }
 
+std::list<NFIQ::QualityFeatureData> NFIQ::NFIQ2Algorithm::computeQualityFeatures(
+  const NFIQ::FingerprintImageData& rawImage )
+{
+	std::list<NFIQ::ActionableQualityFeedback> actionableQuality;
+	std::list<NFIQ::QualityFeatureSpeed> qualityFeatureSpeed;
+  
+	return ( this->pimpl->computeQualityFeatures(
+			rawImage,
+			false, actionableQuality,
```

Comment:
`astyle` defect tagged by Travis CI:
```suggestion
             false, actionableQuality,
```